ai_diffusion/ui/settings.py

```diff
@@ -487,6 +488,32 @@ def __init__(self):
         self.add("debug_dump_workflow", SwitchSetting(S._debug_dump_workflow, parent=self))
         self.add("save_image_metadata", SwitchSetting(S._save_image_metadata, parent=self))
 
+        self.add("save_image_format", ComboBoxSetting(S._save_image_format, parent=self))
+        self.add("save_webp_lossless", SwitchSetting(S._save_webp_lossless, parent=self))
+        self.add("save_image_quality", SpinBoxSetting(S._save_image_quality, self, 0, 100, 1))
+
+        self._widgets["save_image_format"].value_changed.connect(self._update_image_quality_widgets)
+        self._widgets["save_webp_lossless"].value_changed.connect(
+            self._update_image_quality_widgets
+        )
+
+        add_header(self._layout, Settings._save_image_file_name_format)
+        self.add(
+            "save_image_file_name_format",
+            TextSetting(Settings._save_image_file_name_format, parent=self),
+        )
+        self._widgets["save_image_file_name_format"].show_label = False
```

Comment:
I'd prefer to keep this setting editable in the `settings.json` for now (ie. you can keep using it, but without UI). It feels quite niche to me and not so easy to use, and I'm trying to avoid interface settings from growing too much. Probably also not something you constantly need to change.